### PR TITLE
Show people in a region as a bulleted list within the popup

### LINKED ISSUE #133: Show people in a region as a bulleted list within the popup
After fixing #123, it's showing that there are "2 people" but we should be instead showing a list of the people, like:

```
* firstPerson
* secondPerson
```

_Originally posted by @jywarren in https://github.com/publiclab/leaflet-blurred-location-display/pull/123#issuecomment-896154396_

### PR DESCRIPTION
**Pull request by daemon1024**
Fixes #133

### CODE REVIEW

**Comment by daemon1024** [comment]
image
It works 😄

**Comment by daemon1024** [comment]
//offtopic
Btw just noticed, we could use some updates on welcome bot messages. Potential good first issues, across repositories.

File: src/blurredLocationDisplay.js [hunk 1]
```diff
@@ -164,6 +165,8 @@ BlurredLocationDisplay = function BlurredLocationDisplay(options) {
                 var m = L.marker([latitude, longitude], {
                   icon: icon_color
                 }) ;
+                m.title = title;
```
Comment (daemon1024) [inline src/blurredLocationDisplay.js:168]:
```suggestion
                //Need these additional fields to show information in popup 
                m.title = title;
```

**Comment by jywarren** [comment]
> //offtopic
> Btw just noticed, we could use some updates on welcome bot messages. Potential good first issues, across repositories.

great idea. 

OK, and this one is good to merge?

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Looks like it! Thanks!!!

**Comment by jywarren** [comment]
Would you like to add a package.json version increment of `0.0.1`?

**Comment by jywarren** [comment]
and run `npm install` so we get an updated package-lock.json too? Then I'll be able to publish it as a new version on NPM

**Comment by daemon1024** [comment]
Done :)

**Comment by jywarren** [comment]
As soon as this publishes, let's link and follow to the PR in LEL, and then same to plots2 -- sound good? Thank you!!! Also doesn't have to be tonight, it's so late!

**Comment by daemon1024** [comment]
Yup, Sounds good to me. Thank You and Good Night 🌃

**Comment by jywarren** [comment]
Published! Good night!!!

**Comment by jywarren** [comment]
If dependabot doesn't open it, we can open it manually!

**Comment by jywarren** [comment]
Yay! https://github.com/publiclab/leaflet-environmental-layers/pull/562

**Comment by jywarren** [comment]
OK! We more or less got it... still seeing an "undefined" but very close!!! Did we use the wrong property name?

image

https://github.com/publiclab/plots2/pull/10050

**Comment by jywarren** [comment]
Hmm. 

```json
 {
      "doc_id": 768638,
      "doc_type": "PLACES",
      "doc_url": "/profile/charliejohnson",
      "doc_title": "charliejohnson",
      "doc_score": null,
      "doc_author": null,
      "doc_image_url": "https://www.gravatar.com/avatar/6e62cf8e8246c20b6b9839c175dc996a",
      "latitude": "0",
      "longitude": "0",
      "blurred": true,
      "category": "PLACES",
      "place_name": null,
      "created_at": "2021-08-10T11:07:07.000Z",
      "time_since": null,
      "comment_count": null
    }
```

File: src/blurredLocationDisplay.js [hunk 2]
```diff
@@ -151,6 +151,7 @@ BlurredLocationDisplay = function BlurredLocationDisplay(options) {
 
               var id = obj["id"] ;
               var url = obj["url"] ;
+              var title = obj["title"] ;
```
Comment (jywarren) [inline src/blurredLocationDisplay.js:154]: Here, it should work, i think:

https://github.com/publiclab/leaflet-blurred-location-display/blob/86e873954769cc01ef8061e3bfa9237f09276c87/src/blurredLocationDisplay.js#L60

**Comment by daemon1024** [comment]
The url is right, just the title is undefined somehow 🤔

**Comment by jywarren** [comment]
For the screenshot above, I was def. getting `doc_title` values... 

```json
{
  "doc_id": 766513,
  "doc_type": "PLACES",
  "doc_url": "/profile/dennlawgroup",
  "doc_title": "dennlawgroup",
  "doc_score": null,
  "doc_author": null,
  "doc_image_url": "/public/system/users/photos/000/766/513/thumb/denn-law-logo-1x-2.png",
  "latitude": "42",
  "longitude": "-71",
  "blurred": true,
  "category": "PLACES",
  "place_name": null,
  "created_at": "2021-07-21T08:33:27.000Z",
  "time_since": null,
  "comment_count": null
}
```

**Comment by jywarren** [comment]
Aha!!!!

https://github.com/publiclab/leaflet-environmental-layers/blob/702ebe1877ad9d816bd9856798607b325b7ffe13/src/PLpeopleLayer.js#L22-L39

**Comment by jywarren** [comment]
We override the JSONparser function in LEL!

**Comment by daemon1024** [comment]
Ohhh.

**Comment by daemon1024** [comment]
So just one more release then :D Making a PR asap.

**Pull request merged**